drag_and_drop_v2/public/js/view.js

```diff
@@ -116,6 +116,7 @@
                     style: {
                         top: zone.y_percent + '%', left: zone.x_percent + "%",
                         width: zone.width_percent + '%', height: zone.height_percent + "%",
+                        borderStyle: ctx.display_zone_borders ? 'dotted' : 'hidden'
```

Comment:
@agaylard Setting the default border style in `drag_and_drop.css` and changing it here accomplishes what we want, but there's another approach that would be completely compliant with the task description:

1. Introduce an additional CSS class such as `zone-with-borders`:

   ```css
   .xblock--drag-and-drop .zone {
       border: 1px hidden #565656;
   }
   ```

2. Instead of setting the `borderStyle` here, conditionally add `.zone-with-borders` to the "selector" argument that gets passed to the [`h` function](https://github.com/Matt-Esch/virtual-dom/blob/master/virtual-hyperscript/README.md#hselector-properties-children) (`div.zone` in L107 above):

   ```js
   var selector = ctx.display_zone_borders ? 'div.zone.zone-with-borders' : 'div.zone';
   return (
       h(
           selector,
           {  // ...
   ```